roles/ckan/templates/kubernetes/ckan.yaml

```diff
@@ -10,7 +10,7 @@ data:
   ckan-uwsgi.ini: "{{ lookup('template', 'templates/ckan/ckan-uwsgi.ini') | b64encode }}"
   wsgi.py: "{{ lookup('template', 'templates/ckan/wsgi.py') | b64encode }}"
 {% if ckan_googleanalytics_enable %}
-  google_analytics_credentials.json: "{{ ckan_googleanalytics_credentials | b64encode }}"
+  google_analytics_credentials.json: "{{ ckan_googleanalytics_credentials | to_json | b64encode }}"
```

Comment:
On cloud (azure) deploys, this has to NOT do this. 

TODO: investigate locally